blog/terraform-post.md

```diff
@@ -0,0 +1,132 @@
+---
+title: AEP Terraform Provider
+date: 2025-03-31
+authors:
+  - name: Alex Stephen
+  - name: Marsh Gardiner
+---
+
+# AEP Terraform Provider
+
+Author: Alex Stephen, Marsh Gardiner
+
+A core thesis of the AEP project has been that API design constraints can
+facilitate the creation (and maintenance\!) of API client tools. In many
+companies, teams are staffed to build out CLI tools, Terraform providers, UIs,
+LLM agents, and various other tools that act as thin layers on top of a
+company's APIs. The AEP project aims to ensure that these tools can be easily
+generated on top of our API design, ensuring that companies can move faster
+with less overall effort. We’re excited to launch our first version of a
+[fully auto-generated Terraform provider](https://github.com/aep-dev/terraform-provider-aep)
+based off the AEPs.
+
+![][image1]
+
+# What is Terraform?
+
+Terraform is a powerful Infrastructure-as-Code (IaC) tool that allows you to
+define and manage your infrastructure using code. Infrastructure-as-Code, in
+general, is the practice of managing and provisioning infrastructure through
+machine-readable definition files rather than through manual processes. This
+means you can automate the creation, modification, and deletion of resources
+across various cloud providers and on-premises environments, ensuring
+consistency, repeatability, and version control.
+
+Cloud companies often build out Terraform providers, which are responsible for
+establishing differences between the cloud and the user's intent, determining
+which API requests are necessary, making those requests, and handling any
+follow-up actions.
+
+# How do the AEPs help with this?
+
```

Comment:
```suggestion

Most companies must staff engineers just to maintain their Terraform provider. However, making AEP-compliant APIs can give you providers for no additional effort beyond writing the API!
```

Maybe @earth2marsh would massage this into something better - but I think it's important to really sell how much this reduces the work that needs to be done to add Terraform support.